### DISCUSSION TITLE
为1.17及以上版本选择安装forge的同时提示“不推荐安装optifine”

### DISCUSSION THREAD

**Discussion by 3322karry** [Meloong-Git/PCL#2407] [提问] [answered]
因为最近在贴吧有很多因为同时安装forge和optifine的萌新提问，而且一问三不知

**Comment by XiaoLLiu617** [comment]
无用，萌新就是萌新，要有过程，记住不是所有玩家都是萌新都需要这个功能

**Reply by Chiloven945** [reply]
您的意思是说就让萌新自己摸索咯？等同于让你自学义务教育？

**Reply by XiaoLLiu617** [reply]
> 您的意思是说就让萌新自己摸索咯？等同于让你自学义务教育？

我有必要告诉你，不是所有的玩家都是萌新，但这个功能只有萌新需要，萌新本来就不是很懂，你再弹个窗，萌新更懵了，反道事的起反，让萌新摸索慢慢适应是更好的方法，而不是丢给他所有的知识！

**Reply by XiaoLLiu617** [reply]
> 您的意思是说就让萌新自己摸索咯？等同于让你自学义务教育？

pcl2的帮助页面也不是没有，萌新不会开光影还不会自己去找

**Comment by WForst-Breeze** [comment, marked as answer]
交个反馈？

**Comment by Chiloven945** [comment]
实际上安装optifine时就应该提示一下，毕竟很多不兼容的点，无论是forge还是fabric
更改为“安装optifine的注意事项”似乎更为合适，原话有点选边站的意味……
